MCDTool.egg-info/PKG-INFO

```diff
@@ -0,0 +1,14 @@
+Metadata-Version: 2.1
+Name: MCDTool
+Version: 0.0.1
+Summary: A probabilistic Multi Criteria Decision Analysis
+Home-page: UNKNOWN
```

Comment:
We should definitely add a homepage, maybe a subpage on futureForest, or at least a small github page.

Reply:
`Platform` and `Home-page` should still be filled in.
For the "Platform", what do you think we should add here?
For the "Home-page", @mspada and I prefer something general and not related specifically to FutureForest. The GitHub repository link is fine but maybe we should first move the repository into the public part of wetransform @thorsten-reitz?